recipes/bitserializer/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+from conans import ConanFile, tools
+import os
+
+
+class BitserializerConan(ConanFile):
+    name = "bitserializer"
+    description = "Core part of C++ 17 library for serialization to JSON, XML, YAML"
+    topics = ("serialization", "json", "xml")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://bitbucket.org/Pavel_Kisliak/bitserializer"
+    license = "MIT"
+    settings = "compiler"
+    no_copy_source = True
+
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
+        tools.check_min_cppstd(self, "17")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        for dirname in os.listdir(self.source_folder):
+            if "-bitserializer-" in dirname:
+                os.rename(dirname, self._source_subfolder)
+                break
```

Comment:
I am not sure it this will be allowed

```suggestion
        extracted_dir = self.name + "-" + self.version
        os.rename(extracted_dir, self._source_subfolder)
```

Reply:
Ahhh thank you for explaining, might be worth mentioning in your new PR as a comment see what the CCI team say =)

Reply:
Ok, I will add comment